src/functions-and-arrays.js

```diff
@@ -23,16 +56,42 @@ function sum() {}
 
 
 // Iteration #4: Calculate the average
-// Level 1: Array of numbers
+// Level 1: Array of numbers //this is correct but need to add something for null
+
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-function averageNumbers() {}
+function sumOfNumbers(numbers){
+let sum = 0;
+  numbers.forEach((number)=> {
+  sum = sum + number
+})
+  return sum;
+}
+function averageNumbers(numbers) {
+  if (numbers.length === 0) {return null};
```

Comment:
If you write this if in a line it should be done like this:

`if (numbers.length === 0) return null;`

if not:

```
if (numbers.length === 0) {
     return null;
};
```